include/mbedtls/ssl.h

```diff
@@ -851,6 +865,43 @@ typedef void mbedtls_ssl_async_cancel_t( mbedtls_ssl_context *ssl );
 #endif /* MBEDTLS_KEY_EXCHANGE_WITH_CERT_ENABLED &&
           !MBEDTLS_SSL_KEEP_PEER_CERTIFICATE */
 
+#if defined(MBEDTLS_SSL_DTLS_SRTP)
+
+#define MBEDTLS_TLS_SRTP_MAX_KEY_MATERIAL_LENGTH    60
+#define MBEDTLS_TLS_SRTP_MAX_MKI_LENGTH             255
+/*
+ * List of SRTP profiles for DTLS-SRTP
+ */
+typedef enum
```

Comment:
On second look, I'm not sure this enum is necessary, as we might as well use the named IANA constants throughout the code, and `mbedtls_ssl_srtp_profile` could be replaced by `typdef uint16_t mbedtls_ssl_srtp_profile` to still have a type name to indicate the semantics of those numeric values.

Reply:
Thoughts, @gilles-peskine-arm @mpg @jeannotlapin ?

Reply:
Why not define the enum with the same values as the IANA constants?